### ISSUE TITLE
Mock Clipboard Methods

### ISSUE THREAD

**Issue by afaqRns** [react-native-clipboard/clipboard#107]
I'm writing a unit tests for a project and when I write a test for a module which is dependent on this library, I got following error:

> TypeError: Cannot read property 'setString' of undefined

Following is method to test

```
const onCopyPress = () => {
    Clipboard.setString(userData.publickey);

    ToastAndroid.show(t('createWallet.copiedToast'), ToastAndroid.SHORT);
};
```

Following is the calling point:

```
<TouchableOpacity testID={'copy'} onPress={() => onCopyPress()}>
        <Image source={clipboard} />
</TouchableOpacity>
```

Following is test for the above code:

```
const {getByTestId} = render(
      <Provider store={store}>
        <ThemeContextProvider>
          <Drawer {...props} theme={theme} />
        </ThemeContextProvider>
      </Provider>,
 );

let button = getByTestId('copy');

fireEvent.press(button);

try {
      expect(onCopyPress).toHaveBeenCalled();
} catch (e) {
      console.log('ee', e);
}
```

**Comment by umarrasheedmebest** [comment]
@harisbaig100 @Naturalclar can you guys look into the issue, I am also facing same issue while writing the unit test.

Thanks!

**Comment by Naturalclar** [comment]
@afaqRns @umarrasheedmebest thanks for the suggestion! I've published v1.8.4 which includes the mock files along with instructions in the README for how to mock the clipboard module.
